tests/system/feature_store/test_feature_store.py

```diff
@@ -3874,6 +3874,27 @@ def test_ingest_with_validator_from_uri(self, engine):
 
         assert isinstance(df, pd.DataFrame)
 
+    def test_ingest_with_steps_drop_features(self):
+        key = "patient_id"
+        measurements = fstore.FeatureSet(
+            "measurements", entities=[Entity(key)], timestamp_key="timestamp"
+        )
+        measurements.graph.to(DropFeatures(features=[key]))
+        source = CSVSource(
+            "mycsv", path=os.path.relpath(str(self.assets_path / "testdata.csv"))
+        )
+        with pytest.raises(mlrun.errors.MLRunInvalidArgumentError) as ml_run_exception:
+            fstore.ingest(
+                measurements,
+                source,
+                infer_options=fstore.InferOptions.schema() + fstore.InferOptions.Stats,
+                run_config=fstore.RunConfig(local=True),
```

Comment:
I don't think these parameters are needed for the test.

Reply:
solved